Source/CarthageKit/Xcode.swift

```diff
@@ -1211,6 +1225,8 @@ public func buildInDirectory(directoryURL: NSURL, withConfiguration configuratio
 						}
 						return (project, containedSchemes)
 					}
+					// replace the filter and concat with something that accounts for multiple subprojects
+					// cause it errors out if even one of the results is that error...
```

Comment:
That was actually erroneously left in as a note to myself when I first started work on this.